user_guide/common_features_channels.rst

```diff
@@ -3,13 +3,45 @@
 Channels
 ========
 
-Collection of comprehensive customer-related data is a cornerstone of customer relations management.
 
-To represent different sources of customer-related data, OroCRM uses records of the Channel entity ("channels").
+Today, successful businesses usually have more than one sales site. These may be different online stores, 
+business-to-business enterprises, or business-to-customer outlets. Moreover, different customer and sales-related 
+information can be received from various survey-campaigns or membership in clubs, funds, charity events, etc. Gaining a 
+full understanding of all the information you receive from each of these sources, along with all the individual customer 
+information you collect from various other sources, becomes crucial. With OroCRM's Channels you can do both with minimum 
+effort. 
 
-For example, any store, on-line site, social activity, etc. may be represented with its own channel.  
+In OroCRM, each Channel record represents one sources of customer-related data.
+Upon creation, you can choose what kind of customer-related information will be collected from each source (in technical 
+words "what entities will be assigned to this channel"). 
+
+For example, you have two channels of customer related data: your on-line store and your office.
+While they are both related to your customers, the information and customer relations process is quite different.
+ 
+Of course, there are much more details and peculiarities, but if we keep it simple, in he on-line store, somebody comes 
+to the store's website, looks through the items offered, and if interested in 
+buying them puts the items in the cart and orders them, leaving you the contact and shipping details. In many cases the 
+web-store customers can log-in and their profile details will be used. What kind of information do you need to make 
+relations with such customers successful? The most basic things are - what pages have people viewed, what items they 
+have ordered, if there is anything that a customer has added to the cart but hasn't ordered and if there are some 
+complaints after the delivery.
```

Comment:
Edit: For example, during the customer's online journey, they visit the store's website, look through the items offered, and make a purchase it they are interested. In order to to this, they put the item in the cart and make their order by leaving you their contact and shipping details. In many cases, customers can login to use profile details they have previously saved on your store. With all of this in mind, you should ask yourself what kind of information you need in order to have successful relations with your customers. The most basic factors to keep track of are what pages or products they have viewed, what items they have ordered, if there is anything that they have added to their cart but haven't ordered, and if there are any complaints after the delivery.